src/stores/Store.ts

```diff
@@ -192,14 +490,58 @@ export default class Store {
 		this.isSessionsLoading = false;
 	};
 
+	@action
+	replayMessage = async (message: ParsedMessageItem | ActMessageItem, index: number) => {
+		let result: MessageSendingResponse | ActSendingResponse | null = null;
+		switch (this.selectedSchemaType) {
```

Comment:
i think switch statement is irrelevant since we checking message type.
and replayMessage does not change any observable so it's not an action